trade-module/src/main/java/io/edpn/backend/trade/application/domain/Station.java

```diff
@@ -1,33 +1,23 @@
 package io.edpn.backend.trade.application.domain;
 
-import lombok.AllArgsConstructor;
-import lombok.Builder;
-import lombok.Getter;
-import lombok.Setter;
-import lombok.ToString;
+import lombok.With;
 
 import java.time.LocalDateTime;
-import java.util.ArrayList;
 import java.util.List;
 import java.util.UUID;
 
-@Getter
-@Setter
-@ToString
-@AllArgsConstructor
-@Builder
-public class Station {
-    private UUID id;
-    private Long marketId;
-    private String name;
-    private Double arrivalDistance;
-    private System system;
-    private Boolean planetary;
-    private Boolean requireOdyssey;
-    private Boolean fleetCarrier;
-    @Builder.Default
-    private LandingPadSize maxLandingPadSize = LandingPadSize.UNKNOWN;
-    private LocalDateTime marketUpdatedAt;
-    @Builder.Default
-    private List<MarketDatum> marketData = new ArrayList<>();
-}
+@With
+public record Station(
+        UUID id,
+        Long marketId,
+        String name,
+        Double arrivalDistance,
+        System system,
+        Boolean planetary,
+        Boolean requireOdyssey,
+        Boolean fleetCarrier,
+        LandingPadSize maxLandingPadSize,
+        LocalDateTime marketUpdatedAt,
+        List<MarketDatum> marketData
```

Comment:
Previously had @Builder.Default for both maxLandingPadSize, and marketData, now getting null errors, and could potentially start inserting null landingPadSizes instead of UNKNOWN

Reply:
defining a compact constructor within record `Station`, to ensure that the `maxLandingPadSize` field is never null. If a null value is passed, it defaults to `LandingPadSize.UNKNOWN`.